go.mod

```diff
@@ -19,13 +20,16 @@ require (
 	github.com/gobuffalo/envy v1.6.15 // indirect
 	github.com/gogo/protobuf v1.2.0 // indirect
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
+	github.com/golang/protobuf v1.3.1 // indirect
 	github.com/google/btree v1.0.0 // indirect
 	github.com/google/go-github v17.0.0+incompatible
+	github.com/google/go-querystring v1.0.0 // indirect
 	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
+	github.com/google/martian v2.1.0+incompatible
```

Comment:
do we need to introduce this here?

Reply:
good catch. That was a PR feedback from Gerred, but I was supposed to use whatever controller uses and I screw it up :) Now that I see what controller uses (it's a controller-runtime kubernetes log libbrary) I would probably stick with printing out for now and introduce a logging library (like `logrus` maybe?) in a different PR :)